tests/testlistselector.py

```diff
@@ -173,16 +173,17 @@ class Q(param.Parameterized):
             r = param.ListSelector(default=[6])
 
     ##########################
-    # CEBALERT: not sure it makes sense for ListSelector to be set to
-    # a non-iterable value (except None). I.e. I think this first test
-    # should fail.
     def test_default_not_checked_to_be_iterable(self):
-        class Q(param.Parameterized):
-            r = param.ListSelector(default=6)
+        with pytest.raises(
+            ValueError,
+            match=re.escape("ListSelector parameter 'r' only takes list types, not 6."),
+        ):
+            class Q(param.Parameterized):
+                r = param.ListSelector(default=6)
 
     def test_set_checked_to_be_iterable(self):
         class Q(param.Parameterized):
-            r = param.ListSelector(default=6,check_on_set=False)
+            r = param.ListSelector(default=[6],check_on_set=False)
```

Comment:
Looks like previously it was catching a non-iterable value being set, but not when it was the default. Already verified that the error was caught above, so just updated the value here.